compiler-core/src/language_server/tests/snapshots/gleam_core__language_server__tests__action__create_unknown_module_under_dev.snap

```diff
@@ -0,0 +1,6 @@
+---
+source: compiler-core/src/language_server/tests/action.rs
+expression: output
+---
+// --- File operations
+- Create file:///dev/wobble/woo.gleam
```

Comment:
It seems snapshots including paths don't work since they have different output on windows, hence these tests are failing under windows. I'll change from doing snapshots to just normal asserts in code.